docs/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -82,7 +82,7 @@ PLATFORM Configuration Parameters
 If non-default parameters are selected for the variables in this section, they should be added to the ``platform:`` section of the ``config.yaml`` file. 
 
 ``WORKFLOW_MANAGER``: (Default: "none")
-   The workflow manager to use (e.g., "rocoto"). This is set to "none" by default, but if the machine name is set to a platform that supports Rocoto, this will be overwritten and set to "rocoto." If set explicitly to "rocoto" along with the use of the ``MACHINE: "LINUX"`` target, the configuration layer assumes a Slurm batch manager when generating the XML. Valid values: ``"rocoto"`` | ``"none"``
+   The workflow manager to use (e.g., "rocoto"). This is set to "none" by default, but if the machine name is set to a platform that supports Rocoto, this will be overwritten and set to "rocoto." If set explicitly to "rocoto" along with the use of the ``MACHINE: "LINUX"`` target, the configuration layer assumes a Slurm batch manager when generating the XML. Valid values: ``"rocoto"`` | ``"ecflow"`` | ``"none"``
```

Comment:
I didn't know we can generate ecflow workflows now.  Is that really true?  If so, it's great, and another reason to stick with the SRW.

Reply:
Oof! PR #828 added `ecflow` as a valid value to `ush/valid_param_vals.yaml`, and I took the list of valid values from that file, but now that I'm looking at the PR, I'm seeing that it says "The ecFlow option will be added in a separate PR later." @MichaelLueken thoughts on how to handle this? `ecflow` is a valid value insofar as the workflow shouldn't fail to generate due to an invalid value, but it sounds like it's invalid in the sense that using this option won't produce the result users expect. Or have things been updated?

Reply:
@gspetro-NOAA - Unfortunately, the ability to generate an `ecflow` workflow is still not available in the SRW App.  While `ecflow` has been added to the `ush/valid_param_vals.yaml`, without the necessary `ecf` directory, it is impossible to generate `ecflow` workflows at this time.  A note stating that `ecflow` is currently a valid option, but setting this option will fail to produce a functioning workflow should be sufficient.  Alternatively, the `ecflow` option can be removed from the list here.

Reply:
I think it should be removed from the list here and then added back when the actual functionality is also included.  Otherwise, having it in this list is misleading.

Reply:
Ok, I've removed it in the rst file but added a note because there is still the risk that someone looking in `valid_param_vals.yaml` will be confused.

Reply:
Oh shoot, sorry, I was thinking of the `valid_param_vals.yaml` list when I wrote my previous comment.  I meant that's the list we used to control whether something is actually supported.  Can we remove it from there as well?

Reply:
@MichaelLueken Does ecflow need to be in `valid_param_vals.yaml` for the AQM tests to pass? Or for other AQM-related reasons?

Reply:
@gspetro-NOAA - In the current develop branch, `ecflow` isn't used in the AQM WE2E test or either of the `config.aqm.*.yaml` files.  It should be safe to remove `ecflow` from the `valid_param_vals.yaml` file until functionality has been fully introduced into the SRW App.